index.html

```diff
@@ -64,9 +73,11 @@ <h1>
 </section>
 <section class="works" >
     <div class="recent-works">
-        <h2>My Recent </br> Works</h2>
+        <h2 class="mobile-work-header">My Recent </br> Works</h2>
+        <h2 class="desktop-work-header">My Recent Works</h2>
     </div>
-    <div class="myworks-card">
+    <div class="myworks-wrapper">
+        <div class="myworks-card">
```

Comment:
- [ ] from your mobile version the cards are not also consistent with the figma design as shown below please kindly add the required changes and work on the design for each you can do this by properly using the flex properties to style the contents fo each card. :+1: 
- figma design
![l-c-f](https://user-images.githubusercontent.com/89055502/236343920-a8c4e242-1636-4141-9d30-c440d6335c0b.png)
- your design
![l-c-s](https://user-images.githubusercontent.com/89055502/236343944-be74ff7d-26ff-4661-a620-f9b09e1e1b11.png)